support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx

```diff
@@ -46,64 +47,114 @@ export const getSvgIcon = (isUnlocked: boolean): JSX.Element =>
 export const checkListData = ({
 	higherTier,
 	isAustralia,
+	isEmotionalBenefitTestVariant,
 }: CheckListDataProps): CheckListData[] => {
 	const maybeGreyedOutHigherTier = higherTier ? undefined : greyedOut;
 
-	return [
-		{
-			icon: getSvgIcon(true),
-			text: (
-				<p>
-					<span css={boldText}>A regular supporter newsletter. </span>Get
-					exclusive insight from our newsroom
-				</p>
-			),
-		},
-		{
-			icon: getSvgIcon(true),
-			text: (
-				<p>
-					<span css={boldText}>Uninterrupted reading. </span> See far fewer asks
-					for support
-				</p>
-			),
-		},
-		...(isAustralia
-			? [
-					{
-						icon: getSvgIcon(higherTier),
-						text: (
-							<p>
-								<span css={boldText}>
-									Limited-edition Guardian Australia tote bag.{' '}
-								</span>
-								<span css={highlightTextSpacing} />
-								<span css={highlightText}>Offer ends 31 May</span>
-							</p>
-						),
-						maybeGreyedOut: maybeGreyedOutHigherTier,
-					},
-			  ]
-			: []),
-		{
-			icon: getSvgIcon(higherTier),
-			text: (
-				<p>
-					<span css={boldText}>Full access to our news app. </span>Read our
-					reporting on the go
-				</p>
-			),
-			maybeGreyedOut: maybeGreyedOutHigherTier,
-		},
-		{
-			icon: getSvgIcon(higherTier),
-			text: (
-				<p>
-					<span css={boldText}>Ad-free reading. </span>Avoid ads on all your
-					devices
-				</p>
-			),
-			maybeGreyedOut: maybeGreyedOutHigherTier,
-		},
-	];
+	return isEmotionalBenefitTestVariant && !isAustralia
+		? [
+				{
```

Comment:
This ternary is a little hard to read, could the conditional items be spread into the existing array instead? There's a good example on line 122

Reply:
The new set of benefits are entirely different from the old one, that's why we cannot use the spread operator.